hr_payslip_change_state/__manifest__.py

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 # © 2016 - Eficent http://www.eficent.com/
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": "Human Resources Payslip Change State",
     "summary": "Change the state of many payslips at a time",
     "version": "10.0.1.0.0",
     "license": "AGPL-3",
     "category": "Human Resources",
-    "author": "Eficent, Odoo Community Association (OCA)",
+    "author": "Eficent, Odoo Community Association (OCA)"
+              "Serpent Consulting Services Pvt. Ltd.,",
```

Comment:
OCA must be always the last one. But anyway I wouldn't put Serpent as author as what you did here is just a standard migration, and you didn't add any new feature.